common/components/velocity.hpp

```diff
@@ -7,13 +7,13 @@
 
 namespace tec {
 	struct Velocity {
-		Velocity() : linear(0, 0, 0, 0), angular(0, 0, 0, 0) { };
+		Velocity() : linear(0, 0, 0), angular(0, 0, 0) { };
 
-		Velocity(glm::vec4 linear, glm::vec4 angular)
+		Velocity(glm::vec3 linear, glm::vec3 angular)
 			: linear(linear), angular(angular) { };
 
-		glm::vec4 linear;
-		glm::vec4 angular;
+		glm::vec3 linear;
+		glm::vec3 angular;
```

Comment:
I'm not sure why these were glm::vec4 when they only hold 3D vectors, if this was an early optimization, it probably wasn't a good one